study.md

```diff
@@ -49,19 +49,26 @@ unique. Write the migration would be used to create a table satisfying these
 requirements.
 
 ```ruby
-# your response here
+class newPets < ActiveRecord::Migration
```

Comment:
This won't work because you're using `t` as the iteration identifier and then you're using `i` in the body of the block.

```ruby
class CreatePets < ActiveRecord::Migration[5.0]
  def change
    create_table :pets do |t|
      t.string :name, null: false
      t.string :breed
    end
    add_index :pets, :name, unique: true
  end
end
```

or

```ruby
class CreatePets < ActiveRecord::Migration[5.0]
  def change
    create_table :pets do |t|
      t.string :name, null: false
      t.string :breed
      t.index :name, unique: true
    end
  end
end
```